inc/contract.class.php

```diff
@@ -1423,11 +1424,14 @@ static function dropdown($options = []) {
             $values[$group][$data['id']] = $tmp;
          }
       }
-      return Dropdown::showFromArray($p['name'], $values,
-                                     ['value'               => $p['value'],
-                                           'on_change'           => $p['on_change'],
-                                           'display'             => $p['display'],
-                                           'display_emptychoice' => true]);
+      return Dropdown::showFromArray($p['name'], $values, [
+         'value'               => $p['value'],
+         'on_change'           => $p['on_change'],
+         'display'             => $p['display'],
+         'display_emptychoice' => true,
+         'class'               => $p['class'],
+         'width'               => $p['width'],
```

Comment:
looks like `$p['width']` may be undefined, as it has not been added in default options.